docs/README.md

```diff
@@ -0,0 +1,157 @@
+This README gives an overview of how to build and contribute to the documentation of Apache Flink.
```

Comment:
Could you rename the mentions of `Apache Flink` here as appropriate to mention `Stateful Functions` instead?
In general the references in this README needs to be adjusted.

```suggestion
This README gives an overview of how to build and contribute to the documentation of Stateful Functions.
```